lib/glific_web/schema/flow_types.ex

```diff
@@ -106,7 +106,7 @@ defmodule GlificWeb.Schema.FlowTypes do
     end
 
     @desc "Get a flow for this user"
-    field :flow_get, :flow do
+    field :flow_get, :flow_result do
```

Comment:
I did not get this.

Reply:
@pankaj-ag We are using flow result as it has flow and error as keys. And we are using error key to send error message when flow is used by someone else